tendrl/commons/tests/test_logger.py

```diff
@@ -92,6 +98,12 @@ def test_push_event():
     message["priority"] = "error"
     log.push_event()
 
+def test_push_notification():
+    message = init()
+    log = Logger(message)
```

Comment:
When tendrl starts each component in tendrl will create a namespace with attribute name NS with all tendrl commons objects and it own component-specific objects and flows, etcd.

For example, if you are running tendrl-node-agent then NS contains:
   NS.tendrl:  It contains all commons objects and flows
   NS.node_agent: It contains all  node-agent specific objects and flows
To test any file or function in tendrl we need to create some dummy NS with tendrl and component-specific namespace with all objects or flow or some object required by the function which you are going to test.

Here we are actually testing logger function, so it requires only ClusterMessage, NodeMessage, Message, NotifierMessage objects.

So we are creating dummy NS with some dummy objects with these name. In your case for push notification function required NotifierMessage object so got to https://github.com/mlanotte/commons/blob/b7dc00d5fbd5db5d9ad8253342504d3e82638e8e/tendrl/commons/tests/fixtures/cluster_message.py  and NotifierMessage object like

```class NotificationMessage(object):

    def __init__(self, *args, **kwargs):
        pass

    def save(self):
        pass
```

Reply:
then it will work